README.md

```diff
@@ -1,24 +1,23 @@
-AChecker
-========
+# AChecker
 
 AChecker is an automated accessibility checker used to evaluate the accessibility of HTML pages, and help ensure they can be accessed by all individuals, including those with disabilities, using assistive technologies to navigate the Internet.
 
 AChecker live site: http://achecker.ca
 
 What sets AChecker apart from other automated accessibility checkers?
 
-1. Reviewers can interact with the system to make decisions on potential barriers that automated checkers can not determine with certainty.
-2. Choose from a range of accessibility standards to review conformance with various international accessibility requirements.
-3. Design custom accessibility guidelines tailored specifically to your organization
-4. View existing guidelines in AChecker to see exactly what it is reviewing.
-5. Design new accessibility checks and have them added to AChecker.
+- Reviewers can interact with the system to make decisions on potential barriers that automated checkers can not determine with certainty.
+- Choose from a range of accessibility standards to review conformance with various international accessibility requirements.
+- Design custom accessibility guidelines tailored specifically to your organization
+- View existing guidelines in AChecker to see exactly what it is reviewing.
+- Design new accessibility checks and have them added to AChecker.
 
-Installation
-------------
+## Installation
 
-1. Unzip the achecker.tar.gz file into a php enabled, web accessible directory.
-2. Open the installer at AChecker/install
-3. Follow the instruction provided by the installer.
+- Clone this repository
+- Be sure to have Composer [setup on your system/server](https://getcomposer.org/download/)
```

Comment:
After installing PHP Composer, should it be one more step about installing project dependencies using PHP Composer?

Reply:
[composer.lock](https://github.com/VictorAlagwu/AChecker/blob/ACHECKER-6/composer.lock#L25) requires PHP 7 to install Plates library. Is this the lowest required PHP version?

This also reminds me that we should add a system requirement on PHP and MySQL versions to install AChecker.

I received this error message at running `php  composer.phar install` to install dependencies:
```
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
Package operations: 1 install, 0 updates, 0 removals
  - Installing league/plates (v4.0.0-alpha): Downloading (failed)       
Downloading (failed)       
Downloading (failed)    Failed to download league/plates from dist: The "https://api.github.com/repos/thephpleague/plates/zipball/f53f4c1fa4bf307d0f3858a348172ff4faf7669a" file could not be downloaded: SSL operation failed with code 1. OpenSSL Error messages:
error:1407742E:SSL routines:SSL23_GET_SERVER_HELLO:tlsv1 alert protocol version
Failed to enable crypto
failed to open stream: operation failed
    Now trying to download from source
  - Installing league/plates (v4.0.0-alpha): Cloning f53f4c1fa4 from cache
Generating autoload files
```
Is this related to definitions in composer.lock too?

Reply:
Minimun PHP requirement for Plates is PHP 7.0